user/enterprise/user_role_management.md

```diff
@@ -0,0 +1,214 @@
+---
+title: User Role Management
+layout: en_enterprise
+
+---
+
+_Available from Travis Enterprise 3.1_
```

Comment:
Please change to 3.1.0 instead of 3.1

Reply:
Updated version number and included missing screenshots